updater/reports/ReportTeamsLegacy.py

```diff
@@ -0,0 +1,30 @@
+from .ReportDaily import *
+
+# Organizations created before September 2015 might have legacy admin
+# teams. In order to get the best performance out of GHE, these teams
+# should be migrated to the improved permissions model.
+# c.f.ghttps://help.github.com/enterprise/2.11/user/articles/migrating-your-previous-admin-teams-to-the-improved-organization-permissions/
```

Comment:
There seems to be an issue with `c.f.ghttps`. Looks like the `g` was intended to be a space character?